src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx

```diff
@@ -37,51 +63,71 @@ export class ArtworkSidebarPartnerInfo extends Component<
       <Text variant="md">{partner.name}</Text>
     )
   }
-  renderLocations(locationNames) {
-    return (
-      <Text variant="xs" color="black60" pl={1}>
-        {locationNames.join(", ")}
-      </Text>
-    )
-  }
 
-  render() {
-    const { artwork } = this.props
-    const locationNames =
-      artwork &&
-      artwork.partner &&
-      artwork.partner.locations &&
-      artwork.partner.locations.length > 0 &&
-      // @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION
-      filterLocations(artwork.partner.locations)
-
-    return (
-      <>
-        <Spacer mt={2} />
-
-        {this.renderPartnerName()}
+  const handleInquiry = () => {
```

Comment:
checked in the ticket and we're using the same event 👍

Reply:
Good catch! I checked with Peter and corrected the color.